large_image/tilesource/__init__.py

```diff
@@ -51,7 +49,12 @@ def loadTileSources(entryPointName='large_image.source', sourceDict=AvailableTil
     :param entryPointName: the name of the entry points to load.
     :param sourceDict: a dictionary to populate with the loaded sources.
     """
-    for entryPoint in iter_entry_points(entryPointName):
+    try:
+        from importlib.metadata import entry_points
+    except ImportError:
+        from importlib_metadata import entry_points
```

Comment:
If this is now a hard dependency and `importlib.metadata` is a part of stdlib >=3.8, then, in my opinion, this should be with the rest of the imports at the top of the file

Reply:
Okay.  I'll shift it up.